charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
-        {{ if .Values.controllerManager.apiserverSkipVerify -}}
+        {{ if and (.Values.controllerManager.apiserverSkipVerify) (not .Values.useAggregator) -}}
```

Comment:
> I don't know how to do this with helm

ok, then I guess the current change is fine as a quick fix for the current implementation, but I still think that disabling TLS verification in general is a code smell.
we could probably ask @arschles for the help to address this issue with a better solution separately.

I'm fine with merging this PR as it is.